jkube-kit/common/src/test/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+spring.application.name:demoservice
```

Comment:
maybe move this file to `src/test/resources/util/spring-boot-application.properties` so that it remains in the context of the test-suite (we'll probably have more)